javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -106,6 +105,9 @@ public static void installBytebuddyAgent(Instrumentation inst) {
   public static ResettableClassFileTransformer installBytebuddyAgent(
```

Comment:
are you calling this method from tests? I'm trying to understand how this PR is helping you (so that I don't accidentally break you again in the future)

Reply:
yes I call it from tests